agenda-connectors-webapp/src/main/webapp/vue-app/agenda-connectors/google-connector/agendaGoogleConnector.js

```diff
@@ -37,7 +36,6 @@ export default {
       throw new Error('Google connector can\'t be enabled with empty Client API Key or empty Client Secret Key.');
     }
     this.CLIENT_ID = apiKey;
-    this.SECRET_KEY = secretKey;
```

Comment:
secreKey should be removed from the call
In addition, we must ensure that, in the caller, we no more get the secretKey from server side

Reply:
done here https://github.com/exoplatform/agenda/pull/557